BridgeAppSDK/DataArchiving/SBAEncryptionHelper.swift

```diff
@@ -40,5 +40,36 @@ import UIKit
         let certificatePath = NSBundle.mainBundle().pathForResource(sharedAppDelegate.bridgeInfo.certificateName, ofType: "pem")
         return certificatePath
     }
+    
+    class func encryptedDataFilename() -> String {
+        return "encrypted.zip"
+    }
+    
+    class func encryptedDataPathRoot() -> String {
+        return NSTemporaryDirectory()
```

Comment:
Because (a) that’s where those orphaned files currently are and (b) the uploadToBridge method immediately copies the file to be uploaded into the Application Support directory and uploads it from there, so the original doesn’t have to be in a permanent location.

> On Aug 16, 2016, at 10:42 PM, Shannon Young <notifications@github.com> wrote:
> 
> In BridgeAppSDK/DataArchiving/SBAEncryptionHelper.swift <https://github.com/Sage-Bionetworks/BridgeAppSDK/pull/90#discussion_r75065112>:
> 
> > @@ -40,5 +40,36 @@ import UIKit
> >          let certificatePath = NSBundle.mainBundle().pathForResource(sharedAppDelegate.bridgeInfo.certificateName, ofType: "pem")
> >          return certificatePath
> >      }
> > +    
> > +    class func encryptedDataFilename() -> String {
> > +        return "encrypted.zip"
> > +    }
> > +    
> > +    class func encryptedDataPathRoot() -> String {
> > +        return NSTemporaryDirectory()
> Why the temp directory?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/Sage-Bionetworks/BridgeAppSDK/pull/90/files/b8ebbd47b8d75207420c25b8752bb3511d4b35cf#r75065112>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA6JHvxPmn0a1O_HUgU--BEWHogs--JSks5qgp8-gaJpZM4JmEe3>.
>